src/Kunstmaan/MediaBundle/DependencyInjection/KunstmaanMediaExtension.php

```diff
@@ -43,7 +43,7 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('kunstmaan_media.enable_pdf_preview', $config['enable_pdf_preview']);
         $container->setParameter('kunstmaan_media.blacklisted_extensions', $config['blacklisted_extensions']);
         $container->setParameter('kunstmaan_media.web_root', $config['web_root']);
-        $container->setParameter('kunstmaan_media.full_media_path', $config['web_root'] . '%kunstmaan_media.media_path%');
+        $container->setParameter('kunstmaan_media.full_media_path', $config['web_root'] . '%kunstmaan_media.media_path%'); // Not used, deprecate?
```

Comment:
If this is not used we need to deprecate this, maybe it's only possible to have an entry in the upgrade file to warn users. And add a `// NEXT_MAJOR` command so we know that it should be removed

Reply:
Added an entry to the upgrade file